skaffold/src/main/resources/messages/SkaffoldBundle.properties

```diff
@@ -42,3 +42,4 @@ skaffold.override.image.repo.tooltip=<html>The default image repository allows y
 skaffold.no.file.selected.error=Skaffold configuration file is not selected.
 skaffold.invalid.file.error=Skaffold configuration file doesn't exist or invalid.
 skaffold.corrupted.run.settings=Your Skaffold run configuration is corrupted. Please re-create it to fix.
+skaffold.not.on.system.error=Skaffold path in system is incorrect. Install Skaffold. See required dependencies <a href="https://github.com/GoogleContainerTools/google-container-tools-intellij#prerequisites-and-required-dependencies">here</a>:
```

Comment:
nit suggestion:

`Cloud not detect Skaffold installation. Please install Skaffold. See...`